file/print.go

```diff
@@ -0,0 +1,32 @@
+package file
+
+import (
+	"bytes"
+	"go/ast"
+	"go/format"
+	"go/printer"
+	"go/token"
+)
+
+type RepairInfo struct {
+	Lines []int
+}
+
+func PrintRepair(repaired *ast.File, repairInfo RepairInfo) ([]byte, error) {
+	fset := token.NewFileSet()
+	start := repaired.Pos()
+	end := repaired.End()
+	fset.AddFile(`doesntmatter.go`, int(start), int(end-start+1)).SetLines(repairInfo.Lines)
```

Comment:
`doesntmatter.go` is a file name for the `token.FileSet` - its used internally to resolve positions, etc. But, since I'm creating the file set at line 16 and only ever adding one file to it - its name does not matter. Hence `doesntmatter.go`. I believe this could also be an empty string without issue.